src/IMDb.Infrastructure/Repositories/NameBasicsRepository.cs

```diff
@@ -113,5 +79,10 @@ public async Task<IReadOnlyList<NameBasics>> FindByTitle(int page, string tconst
                 .Take(pagination.Take)
                 .ToListAsync();
         }
+
+        protected override string GetUrl()
```

Comment:
Same thing with this, change them all to be: `protected override string GetUrl() => "https://etc";`